gradle/dependencies.gradle

```diff
@@ -6,8 +6,11 @@ dependencies {
     testImplementation 'org.mockito:mockito-inline:5.2.0'
     testImplementation 'io.opentelemetry:opentelemetry-api:1.37.0'
     testImplementation 'io.opentelemetry:opentelemetry-context:1.37.0'
+    testImplementation 'io.opentelemetry.semconv:opentelemetry-semconv:1.25.0-alpha'
+    testImplementation 'io.opentelemetry.semconv:opentelemetry-semconv-incubating:1.25.0-alpha'
     testImplementation 'io.github.std-uritemplate:std-uritemplate:0.0.55'
-    
+
+    implementation 'com.google.guava:guava:33.1.0-jre'
```

Comment:
we worked hard to remove the dependency on guava, can you explore alternatives so we don't bring it back please?

Reply:
Removed it for this PR to focus on fixing LFU response handling.
Logging a separate issue to streamline error handling when the core library is used across the languages. Will link here.

Reply:
Issue logged for future consideration https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/issues/826